### PR TITLE
Remove direct/indirect import and export distinction, and simplify text.

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -496,14 +496,16 @@ using EAP for example.
...
+Either the handshake protocol or the application directly can supply pre-shared keys for use
+in encrypting (and authenticating) connections to a peer.
+  - TLS
+  - DTLS
+  - QUIC
+  - ESP
+  - OpenVPN
+  - tcpcrypt
+  - MinimalT
+  - WireGuard
```
Comment (tfpauly) [inline draft-ietf-taps-transport-security.md:509]: Let's add IKEv2 here too

**Review by tfpauly — APPROVED** [review]

File: draft-ietf-taps-transport-security.md [hunk 2]
```diff
@@ -496,14 +496,17 @@ using EAP for example.
...
-Either the handshake protocol or the application directly can supply pre-shared keys for the
-record protocol use for encryption/decryption and authentication. If the application can supply
-keys directly, this is considered explicit import; if the handshake protocol traditionally
-provides the keys directly, it is considered direct import; if the keys can only be shared by
-the handshake, they are considered non-importable.
-  - Explicit import: QUIC, ESP, OpenVPN
-  - Direct import: TLS, DTLS, tcpcrypt, MinimalT, WireGuard
-  - Non-importable: CurveCP
+Either the handshake protocol or the application directly can supply pre-shared keys for use
+in encrypting (and authenticating) connections to a peer.
```
Comment (renghardt) [inline draft-ietf-taps-transport-security.md]: Maybe let's substitute "connections" here with "communications" or "data sent"? Not all of these protocols are connection-oriented.

**Review by renghardt — APPROVED** [review]
Thanks, looks good! 
Just suggesting one tweak in the wording for the import text.

**Comment by renghardt** [comment]
Oh, and just double-checking:
For PSK Export, SRTP and QUIC are a "No"? Because the old text said "TLS (for QUIC), DTLS (for SRTP)".

**Comment by tfpauly** [comment]
Yeah, I think that since QUIC and SRTP themselves don't export, but just use the export from inner protocols, this makes sense as is.

File: draft-ietf-taps-transport-security.md (hunk 2 above)
Comment (tfpauly) [reply draft-ietf-taps-transport-security.md]:
```suggestion
in encrypting (and authenticating) communication with a peer.
```

**Pull request merged**